ament_tools/build_types/cmake.py

```diff
@@ -167,8 +167,9 @@ def _common_cmake_on_build(self, should_run_configure, context, prefix, extra_cm
             solution_file = solution_file_exists_at(
                 context.build_space, context.package_manifest.name)
             cmd = prefix + [MSBUILD_EXECUTABLE]
-            if '-j1' in context.make_flags:
+            if not '-j1' in context.make_flags:
                 cmd += ['/m']
+                os.environ['CL'] = '/MP'
```

Comment:
As always Windows is not making it easier. While on other platforms you can just call `VAR=VALUE cmd` in Windows this requires some extra hoops. This function (https://github.com/ament/ament_tools/blob/51add3f0c449db97d63dc32a99c09fab495af728/ament_tools/build_types/cmake.py#L389) generates a batch file which sets up some environment and then calls whatever was passed to it (`%*`).

So either this file gets extended to contain `CL` or another file setting only `CL` is generated which is then nested.